packages/hub/config/default.cjs

```diff
@@ -13,6 +13,10 @@ module.exports = {
       region: 'ap-southeast-1',
       roleChain: ['prod:storage-bucket-writer-role'],
     },
+    ses: {
+      supportEmail: 'support@cardstack.com', // the support email is listed on both staging and prod as a verified SES identity
+      region: 'us-east-1', // staging uses us-east-1. prod uses eu-west-1
```

Comment:
I didn't hardcode it because it feels like something that should be externalised, but I am unsure if that is true? 

I think it should be externalised because it is easier to see the infra for folks who specialise in it and/or don't really work with hub that much, without having to understand the `config` package and what we are doing with it.